consensus/quorum/one-node-one-vote.go

```diff
@@ -21,6 +25,15 @@ func (v *uniformVoteWeight) Policy() Policy {
 	return SuperMajorityVote
 }
 
+// AddNewVote ..
+func (v *uniformVoteWeight) AddNewVote(
+	p Phase, pubKeyBytes shard.BLSPublicKey,
```

Comment:
Name here can be a little frustrating. Naming with `Bytes` looks like a byte slice to me since built-in module `bytes` is the module of byte slice. Can we just name it as `pubKey`?